software/spmd/nprimes/Makefile

```diff
@@ -1,19 +1,3 @@
-#########################################################
-# Network Configutaion
-# If not configured, Will use default Values
-#	bsg_global_X ?= $(bsg_tiles_X)
-#	bsg_global_Y ?= $(bsg_tiles_Y)+1
-bsg_global_X = 4
-bsg_global_Y = 4
-
-#########################################################
-#Tile group configuration
-# If not configured, Will use default Values
-#	bsg_tiles_org_X ?= 0
-#	bsg_tiles_org_Y ?= 1
-bsg_tiles_org_X =1
-bsg_tiles_org_Y =1
-
 # If not configured, Will use default Values
 #	bsg_tiles_X ?= 2
 #	bsg_tiles_Y ?= 2
```

Comment:
is nprimes supposed to run on a 1x1 tile group, or a 2x2 tile group?

Reply:
1x1

Reply:
But the default is 4x4 and tiles_X tiles_Y isn't specified?